proto/frequenz/api/dispatch/v1/dispatch.proto

```diff
@@ -381,7 +381,7 @@ message UpdateMicrogridDispatchRequest {
       repeated RecurrenceRule.Weekday byweekdays = 6;
 
       // On which day(s) of the month does the event occur
-      repeated uint32 bymonthdays = 7;
+      repeated int32 bymonthdays = 7;
```

Comment:
Sorry to be a PITA, but I would expand the docs here too, like you did in the previous PR:

```
  // On which day(s) of the month does the event occur. Valid values are 1 to 31 or -31 to -1.
  //
  // For example, -10 represents the tenth to the last day of the month.
  // The bymonthdays rule part MUST NOT be specified when the FREQ rule part is set to WEEKLY.
```